test/WebJobs.Extensions.Tests/Extensions/NotificationHubs/NotificationHubAsyncCollectorTests.cs

```diff
@@ -41,12 +43,15 @@ public class NotificationHubAsyncCollectorTests
             mockNhClientService.Setup(x => x.SendNotificationAsync(notification, "foo||bar"))
                     .Returns(Task.FromResult(new NotificationOutcome()));
 
-            IAsyncCollector<Notification> collector = new NotificationHubAsyncCollector(mockNhClientService.Object, "foo||bar");
+            TestTraceWriter trace = new TestTraceWriter(TraceLevel.Info);
+
+            IAsyncCollector<Notification> collector = new NotificationHubAsyncCollector(mockNhClientService.Object, "foo||bar", true, trace);
```

Comment:
It looks like this is testing the scenario where `notificationOutcome.Results` is null. You should have a test where there are actual results there as well to make sure that output works as expected. And as I mentioned elsewhere, also test that you write something out when Results is not null but the Results collection is empty.